ivy/functional/backends/tensorflow/__init__.py

```diff
@@ -148,6 +150,45 @@
 
 native_inplace_support = False
 
+backend = "tensorflow"
+
+
+def _handle_inplace_mode():
+    if not hasattr(ivy, "inplace_mode") or ivy.inplace_mode not in [
+        "lenient",
+        "strict",
+    ]:
+        ivy.set_inplace_mode("lenient")
+
+    if ivy.inplace_mode == "lenient":
+        message = (
+            f"The current backend: '{backend}' does not support "
+            "inplace updates natively for arrays but does for variables. "
+            "And since the current ivy.inplace_mode is set to 'lenient', "
+            "Ivy would quietly create new arrays when using inplace "
+            "updates, but a 'strict' mode is available if you want to "
+            "control your memory management and raise an error "
+            "whenever an inplace update is attempted."
+        )
+    else:
+        message = (
+            f"The current backend: '{backend}' does not support "
+            "inplace updates natively for arrays but does for variables. "
+            "And since the current ivy.inplace_mode is set to 'strict', "
+            "Ivy will throw an error "
+            "whenever an inplace update is attempted."
+        )
+    warnings.warn(message)
+    warnings.warn(
```

Comment:
Let's include the information about views in the same warning as mentioned above